src/js/index.js

```diff
@@ -1,3 +1,52 @@
 import '../scss/main.scss';
+// import '../about.html';
```

Comment:
It's not necessary to import de HTML file here. If you create the link to the page from the `index.html`, `parcel` compile the pages automatically.